--- apertium-es-an-0.2.0/Makefile.am.orig	2011-03-21 20:31:35.000000000 +0100
+++ apertium-es-an-0.2.0/Makefile.am	2011-09-21 20:13:13.187088467 +0200
@@ -178,17 +178,15 @@
 	trules-$(PREFIX2).xml  \
 	trules-$(PREFIX2).bin \
 	trules-$(PREFIX1).xml  \
-	trules-$(PREFIX1).bin \
-	$(PREFIX1).mode \
-	$(PREFVAR1)$(PREFIX2).mode 
+	trules-$(PREFIX1).bin
 
 install-data-local:
 	apertium-gen-modes modes.xml apertium-$(PREFIX1)
 	cp *.mode modes/
-	$(INSTALL_DATA) $(PREFVAR1)$(PREFIX2).mode $(apertium_es_modesdir)
-	$(INSTALL_DATA) $(PREFIX1).mode $(apertium_es_modesdir)
-	$(INSTALL_DATA) $(BASENAME).trules-$(PREFIX1).xml $(apertium_es_andir)
-	$(INSTALL_DATA) $(BASENAME).trules-$(PREFIX2).xml $(apertium_es_andir)
+	$(INSTALL_DATA) $(PREFVAR1)$(PREFIX2).mode $(DESTDIR)$(apertium_es_modesdir)
+	$(INSTALL_DATA) $(PREFIX1).mode $(DESTDIR)$(apertium_es_modesdir)
+	$(INSTALL_DATA) $(BASENAME).trules-$(PREFIX1).xml $(DESTDIR)$(apertium_es_andir)
+	$(INSTALL_DATA) $(BASENAME).trules-$(PREFIX2).xml $(DESTDIR)$(apertium_es_andir)
 
 CLEANFILES = -rf $(TARGETS) modes .deps/ 
 
